src/sentry/static/sentry/app/views/teamMembers.jsx

```diff
@@ -25,8 +25,7 @@ const TeamMembers = React.createClass({
   componentWillReceiveProps(nextProps) {
     let params = this.props.params;
     if (
-      nextProps.params.teamId !== params.teamId ||
-      nextProps.params.orgId !== params.orgId
+      nextProps.params.teamId !== params.teamId || nextProps.params.orgId !== params.orgId
```

Comment:
This looks like you ran an old version of Prettier, because 1.2.2 handles this case but 1.0.0 didn't.